artifact/v3/scripts/executor.cpp

```diff
@@ -270,6 +265,22 @@ Error ScriptRunner::AsyncRunScripts(State state, Action action, HandlerFunction
 	return Execute(scripts_iterator, scripts_iterator_end, handler);
 }
 
+
+Error ScriptRunner::RunScripts(State state, Action action) {
+	auto run_err {error::NoError};
+	auto err {AsyncRunScripts(state, action, [this, &run_err](Error error) {
```

Comment:
Same here wrt using explicit `=`. But again, I might be too late and this is now all across our code base?

Reply:
Right. By looking at the other PR from @kacf I can see that this is [common place](https://github.com/mendersoftware/mender/pull/1366/commits/42455fe465df6e47fd3c9f1c93f97680682b3f92#diff-cfe2c3235321dfa58992c067e1a040136acfb859fd842096917ce75a9209f58aR56). Too late, then.

Reply:
Initialization wasn't really the point I was trying to make though. I fully agree with that, based on the style guide and subtle effects people have described about it.

The thing which triggered me was that much more than initialization is being done here. `AsyncRunScripts` is full of side effects.

So it's more about readability and expectations, but maybe I'm just being old fashioned and being used to the `err = SomeBigOperation()`. Since you disagree and there are valid reasons for brace initialization, I won't object to it (and might even use it, we'll see 🙂).

Reply:
That is the distinction that I was also trying to make (in my other comment): out of the constructors world, it just reads so much better to use the `=` sign.

(I also feel I am being old fashioned here :sweat: )